tests/data_context/store/test_store_backends.py

```diff
@@ -195,47 +195,47 @@ def test_StoreBackend_id_initialization(tmp_path_factory):
     )
 
     # TODO: Fix GCS Testing
-    # TupleGCSStoreBackend
```

Comment:
Are you intending to have this remain commented-out?

Reply:
I don't feel great about it. Let me take another crack at this to see if I can get a reasonable TupleGCSStoreBackend test.

Reply:
I couldn't come up with a test that didn't mock so much of GCS that we would essentially be doing an "assert value_i_set == value_i_set". In the future perhaps we can implement a docker GCS emulator like https://github.com/fsouza/fake-gcs-server But I'm happy to discuss if there is a better way in the near term.